core/src/main/java/io/questdb/griffin/RecordToRowCopierUtils.java

```diff
@@ -622,6 +627,9 @@ public static RecordToRowCopier generateCopier(
                         case ColumnType.STRING:
                             asm.invokeInterface(wPutStr, 2);
                             break;
+                        case ColumnType.UUID:
+                            asm.invokeInterface(wPutUuidStr, 2);
```

Comment:
there should be ImplicitCastException thrown if UUID cannot be cast to string, allocating (and introducing) a new exception is not advisable here

Reply:
done